spec.bs

```diff
@@ -223,7 +223,38 @@ The <dfn for="interest group">estimated size</dfn> of an [=interest group=] |ig|
 
 <h2 id="leaving-interest-groups">Leaving Interest Groups</h2>
 
-TODO: add this section
+In order to remove a user from a particular interest group, the
+{{Window/navigator}}.{{Navigator/leaveAdInterestGroup()}} can be called.
+
+TODO: Edit the following from the explainer. As a special case to support in-ad UIs, invoking
+navigator.leaveAdInterestGroup() from inside an ad that is being targeted at a particular interest
+group will cause the browser to leave that group, irrespective of permission policies. Note that
+calling navigator.leaveAdInterestGroup() without arguments isn't supported inside a component ad
+frame.
+
+<xmp class="idl">
+[SecureContext]
+partial interface Navigator {
+  Promise<undefined> leaveAdInterestGroup(AuctionAdInterestGroupKey group);
+};
+
+dictionary AuctionAdInterestGroupKey {
+  required USVString owner;
+  required USVString name;
+};
+</xmp>
+
+The <dfn for=Navigator method>leaveAdInterestGroup(group)</dfn> method steps
+are:
+
+1. Let |p| be [=a new promise=].
+1. TODO... the rest (referencing |p| here so the build tool isn't sad).
+
+<h2 id="permission-delegation">Permission Delegation</h2>

```

Comment:
I don't know where this should go, but it seems like something worth speccing.

Reply:
https://github.com/WICG/turtledove/pull/508 added this.

Reply:
Cool, let me delete!